### PR TITLE
Google.EarthPro version 7.3.4

### PR DESCRIPTION
**Pull request by DW-42**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36939)

### CODE REVIEW

File: manifests/g/Google/EarthPro/7.3.4/Google.EarthPro.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Google.EarthPro
+PackageVersion: 7.3.4
```
Comment (OfficialEsco) [inline manifests/g/Google/EarthPro/7.3.4/Google.EarthPro.installer.yaml:5]:
```suggestion
PackageVersion: 7.3.4.8248
```

File: manifests/g/Google/EarthPro/7.3.4/Google.EarthPro.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Google.EarthPro
+PackageVersion: 7.3.4
```
Comment (OfficialEsco) [inline manifests/g/Google/EarthPro/7.3.4/Google.EarthPro.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.3.4.8248
```

File: manifests/g/Google/EarthPro/7.3.4/Google.EarthPro.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Google.EarthPro
+PackageVersion: 7.3.4
```
Comment (OfficialEsco) [inline manifests/g/Google/EarthPro/7.3.4/Google.EarthPro.yaml:5]:
```suggestion
PackageVersion: 7.3.4.8248
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Comment by DW-42** [comment]
Wrong format for the version number

**Pull request closed without merging**